schema/base.graphqls

```diff
@@ -19,6 +31,11 @@ type SignalAggregations {
 
 type SignalCollection {
   tokenID: Int!
```

Comment:
Should this be `tokenId`? I just try to do the most JavaScripty thing.

Reply:
I don't know why I included this since the tokenId is in the parameter.